src/layouts/tabular/tabular.vue

```diff
@@ -78,14 +69,26 @@
 			@update:sort="onSortChange"
 		>
 			<template #footer>
-				<div class="pagination" v-if="totalPages > 1">
-					<v-pagination
-						:length="totalPages"
-						:total-visible="5"
-						show-first-last
-						:value="page"
-						@input="toPage"
-					/>
+				<div class="footer">
+					<div class="pagination">
+						<v-pagination
+							v-if="totalPages > 1"
+							:length="totalPages"
+							:total-visible="7"
+							show-first-last
+							:value="page"
+							@input="toPage"
+						/>
+					</div>
+
+					<div class="per-page">
+						<span>{{ $t('layouts.tabular.per_page') }}</span>
```

Comment:
Per page is so generic, lets use the translation from `index.json` instead

```suggestion
						<span>{{ $t('per_page') }}</span>
```

Reply:
OK! Gotta figure out how to use the scoped translations....